controllers/topics-controller.js

```diff
@@ -0,0 +1,16 @@
+const app = require('../app');
+const { fetchTopics } = require('../models/model.js');
+// const { handleServerErrors } = require('./error-handling-controllers')
+
+const getTopics = (req, res, next) => {
+    return fetchTopics()
+    .then(topics => {
+        res.status(200).send({topics})
+    })
+    // .catch(err => {
+    //     handleServerErrors(err, req, res, next)
```

Comment:
Here rather than trying to invoke your controller we would normally expect you to invoke next with the error